onboard/catkin_ws/src/sonar/scripts/sonar.py

```diff
@@ -131,14 +132,13 @@ def get_biggest_byte(self, angle):
         split_bytes = [data[i:i+1] for i in range(len(data))]
         filteredbytes = split_bytes[self.FILTER_INDEX:]
         best = np.argmax(filteredbytes)
-              #(index, value)
         return (best+self.FILTER_INDEX, filteredbytes[best])
 
-    def angle_to_radian(self, angle):
-        """Converts gradians to degrees 
+    def gradians_to_radians(self, angle):
```

Comment:
@Will-Denton I assume this was supposed to be gradians to radians, is this correct?

Reply:
resolved - yes